Makefile.common

```diff
@@ -245,10 +245,15 @@ common-docker-publish: $(PUBLISH_DOCKER_ARCHS)
 $(PUBLISH_DOCKER_ARCHS): common-docker-publish-%:
 	docker push "$(DOCKER_REPO)/$(DOCKER_IMAGE_NAME)-linux-$*:$(DOCKER_IMAGE_TAG)"
 
+PROM_LATEST_VERSION = $(shell cat VERSION)
+DOCKER_IMAGE_TAG_MAJOR = $(firstword $(subst ., ,$(DOCKER_IMAGE_TAG)))
```

Comment:
I think what we're looking for is this:

```suggestion
DOCKER_MAJOR_VERSION_TAG = $(firstword $(subst ., ,$(shell cat VERSION)))
```

Reply:
Simplified it to this suggestion and removed the check for a match between the image tag and `VERSION`.